package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "steal-qunit",
+  "name": "my-steal-qunit",
```

Comment:
Yes, this has to be removed. 
I have created this a new package so that I can test support of qunit1 and qunit2 with func unit.
here is the PR
https://github.com/bitovi/funcunit/pull/203/files